docs/user/quick-start.md

```diff
@@ -151,15 +165,67 @@ See [_Config file glossary_][11] for detailed information.
 
 #### Start bot
 
-<!-- #TODO @jwflory -->
+**NOTE**:
+This section is one opinionated way to start and configure TeleIRC.
+Experienced system administrators may have other preferences and slight deviation is permittable.
+However _upstream only offers free support for installations that follow our documentation_.
+
+To start the bot, you need to consider the following factors:
+
+1. Where will the binary go?
+1. Where is your config file on the system?
+1. How will you automate the bot to start-up automatically after a system reboot?
+
+##### Example Linux setup
+
+**NOTE**:
+_Looking for an easier way?_
```

Comment:
Is this really an easier way? Would it be worthwhile to simply state that `go run cmd/teleirc.go`, or if built, `./teleirc` will work just as well?

Reply:
@Tjzabel This is the setup guide I would want for myself. It is probably more simple to use `go run cmd/teleirc.go` but I would rather encourage best practices. Plus I was starting to think about packaging this in Fedora/EPEL at some point in the future, so eventually I could deprecate this in support of something like package manager steps.

I'm not convinced the way I wrote it is best, but I'd like to try it out for now and see if folks find it useful or helpful.